### DISCUSSION TITLE
L&A Plasma Carbine

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#36] [Card Balance] [answered]
EQUIPMENT FIREARM
S +2

**Comment by Sol-au-fein** [comment]
Plasma Carbine fire explosive plasma bolts.

Suggestion:
EQUIPMENT FIREARM
S +3. 
Warriors wounded by Plasma Carbine suffers -2/-2/-2 pernament penalty to minimum of 1.

**Reply by Michalx11** [reply]
Another suggestion 
+1 to S if opponnent is wounded or killed draw a card

**Reply by Sol-au-fein** [reply]
I imagine being hit by plasma explosion splash will cover your body, thus penalty to all stats when injured.

**Reply by Michalx11** [reply]
I like both ideas. I always wanted at least one weapon to have ability to draw cards. Carbine is a good candidate because it shots streaks of plasma covering more ground. Damage can be gr ea after and often unexpected but always opening new opportunities on battle grounds... ergo draw a card. ... could be very powerful weapon in the arms of right soldier

**Reply by Sol-au-fein** [reply]
I dont understand your idea im afraid. Look, you are replying to my suggestion with completely different suggestion. I dont think it fits to this weapon.

**Reply by Michalx11** [reply]
Whenever you open discussion here in github it has to do with that card. So its ok to suggest multiple ideas. I actually suggested this idea for Carbine a while back. Just resurfacing it again. I like your suggestion as well but I am adding mine as another option plus I really think both ideas would work with this weapon

**Reply by Sol-au-fein** [reply]
Of course, you absolutely can suggest your idea. Everyone can. But you are replying directly to my suggestion. Do you see Ric's suggestion? It is separated from mine. Nevermind.

**Reply by Michalx11** [reply]
Lol I think you are too technical. But I can see your point... i guess i wasn't paying attention

**Comment by 21-ric** [comment, marked as answer]
@Sol-au-fein based on your suggestion:
EQUIPMENT FIREARM
S +3.
Warriors wounded by Plasma Carbine suffers -1/-1/-1 pernament penalty at each beggining of your opponent turn unless healed (if killed, points are earned).

**Reply by Sol-au-fein** [reply]
Love It!
It would make this weapon really useful.

**Reply by TomekPL-1** [reply]
I like this suggestion

**Comment by Michalx11** [comment]
+1 to S when opponnent is wounded or killed draw a card

**Comment by framerate** [comment]
Going with highlighted answer, but MIGHT not be this patch (need to add ONUNWOUND stat modifier type)
